player/command.c

```diff
@@ -4052,7 +4064,8 @@ static const char *const *const mp_event_property_change[] = {
       "secondary-sub-text", "audio-bitrate", "video-bitrate", "sub-bitrate",
       "decoder-frame-drop-count", "frame-drop-count", "video-frame-info",
       "vf-metadata", "af-metadata", "sub-start", "sub-end", "secondary-sub-start",
-      "secondary-sub-end", "video-out-params", "video-dec-params", "video-params"),
+      "secondary-sub-end", "video-out-params", "video-dec-params", "video-params",
```

Comment:
Non dyn params shouldn't be in this list anymore?

Reply:
They should, they will be checked when notifying clients anyway.